litellm/integrations/prometheus.py

```diff
@@ -789,7 +789,7 @@ def set_llm_deployment_failure_metrics(self, request_kwargs: dict):
             _litellm_params = request_kwargs.get("litellm_params", {}) or {}
             litellm_model_name = request_kwargs.get("model", None)
             model_group = standard_logging_payload.get("model_group", None)
-            api_base = standard_logging_payload.get("api_base", None)
+            api_base = self._strip_trailing_slash(standard_logging_payload.get("api_base", None))
```

Comment:
this should just be done on the standard_logging_payload

Reply:
Looks like you have refactored and pushed the change. Thanks for taking the fix.